jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1521,19 +1052,6 @@ private MethodHandle filterArgumentsPlaceHolder(int index, int parentOffset, int
 		return this;
 	}
 	
-	/*[IF ]*/
-	/*
-	 * Used to preserve the new objectRef on the stack when avoiding the call-in for
-	 * constructorHandles.  Must return 'this' so stackmapper will keep the object
-	 * alive.
-	 */
-	/*[ENDIF]*/
-	@SuppressWarnings("unused")
-	@VMCONSTANTPOOL_METHOD
-	private static Object constructorPlaceHolder(Object newObjectRef) {
```

Comment:
There are a *lot* of `xxxxPlaceHolder` methods.  Why is only this one being moved?

Reply:
Other `xxxxPlaceHolder` methods are non-static. Similar JCL init issues were not seen with non-static MH methods. So, only the static `MH.constructorPlaceHolder` is being moved.